outputs.tf

```diff
@@ -61,3 +61,11 @@ output "igw_id" {
 output "default_network_acl_id" {
   value = "${aws_vpc.mod.default_network_acl_id}"
 }
+
+output "vpc_s3_endpoint_id" {
```

Comment:
Change name to be similar to resource - `vpc_endpoint_s3_id`.

Reply:
updated names to match resource order